src/CoreWCF.Http/tests/AsyncNetAdoptionTests.cs

```diff
@@ -45,6 +45,9 @@ public async Task OneWayPatternTest()
         [InlineData(1000)]
         public async Task OneWayPatternTest_Parallel(int callCount)
         {
+            // Does nothing on .NET [Core], but enables .NET Framework to have sufficient concurrency
+            // to make all the requests
+            System.Net.ServicePointManager.DefaultConnectionLimit = callCount;
```

Comment:
Should probably reset this after the test otherwise you might run into problems where you implicitly fix something in another test with a largely unreasonable value for allowed concurrent connections.

Reply:
You're correct, that would be best practice, I'll fix it. I don't think it's something worth being concerned about in the CoreWCF case though as this only applies for .NET Framework and not .NET [Core]. On .NET the SocketsHttpHandler doesn't limit the connections to 2 like HttpWebRequest does so the WCF client won't hit this problem when running tests on .NET 6/.NET 7 so an implicitly fixed test won't see any failures on .NET 6/.NET 7 anyway.

Reply:
I just realized there is a risk if there's another test running concurrently which changes this value between this test changing it and before the first request (when the value is snapshotted by ServicePointManager when it creates the ServicePoint). We don't currently run our tests concurrently, but that's something I do wish to fix at some point to speed up the test run. As this is the only test doing this, it's not a problem, but if we add other tests doing this and start executing tests concurrently, we could hit a problem. Still going to revert it here though as it's a potential future problem and not a current one.